tests/share/test_sources.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from furl import furl
+import pendulum
+import requests
+from requests.exceptions import Timeout
+from sickle import Sickle
+
+from share.models import SourceConfig
+
+
+@pytest.mark.django_db
+class TestSources:
```

Comment:
Should these tests be disabled by default? I think it's great we have them, but I suspect they take a while and it doesn't seem necessary to run them every time I make a PR.